src/components/CelTemplateSyntaxExplanation.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+const CelTemplateSyntaxExplanation = () => {
+	const templateSyntax = "${}";
+
+	return (
+		<>
+			<p>
+				Variables, macros, and expressions can be interpolated into your
```

Comment:
I think this should maybe read as "The results of CEL expressions can be interpolated into your policy's config...For a complete list of available variables and functions or to see a more detailed explanation, checkout the docs..."